packages/central-server/src/apiV2/utilities/constructNewRecordValidationRules.js

```diff
@@ -446,26 +446,45 @@ export const constructForSingle = (models, recordType) => {
         survey_id: [constructRecordExistsWithId(models.survey)],
         assignee_id: [constructIsEmptyOr(constructRecordExistsWithId(models.user))],
         due_date: [
-          (value, { status }) => {
-            if (status !== 'repeating' && !value) {
-              throw new Error('Due date is required for non-recurring tasks');
+          (value, { repeat_schedule: repeatSchedule }) => {
+            if (repeatSchedule) {
+              if (value) {
+                throw new Error('Non-recurring tasks must not have a due date');
```

Comment:
```suggestion
                throw new Error('Recurring tasks must not have a due date');
```
I assume this is a typo?